include/core/federated/network/net_common.h

```diff
@@ -483,7 +483,6 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   do {                                                                                                                 \
     buffer[0] = MSG_TYPE_STOP_REQUEST;                                                                                 \
     encode_int64(time, &(buffer[1]));                                                                                  \
-    assert(microstep >= 0);                                                                                            \
```

Comment:
Curious about this change. Why safe to remove?

Reply:
Our microstep type is unsigned. I guess there is the possibility that someone pass in an integer though, since it is only a macro. I will add the assertion bacl